src/utils.js

```diff
@@ -0,0 +1,48 @@
+const SIGN_INPUT_ERROR_CLASSNAME = "sign-input-error";
+const ERROR_MESSAGE_CLASSNAME = "error-message-on";
+const EMAIL_REGEX = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g;
+const PASSWORD_REGEX_STRING = /[a-zA-Z]/;
+const PASSWORD_REGEX_NUMBER = /\d/;
+
+export function setInputError(elements, message) {
+  elements.input.className += ` ${SIGN_INPUT_ERROR_CLASSNAME}`;
+  elements.errorMessage.className += ` ${ERROR_MESSAGE_CLASSNAME}`;
+  elements.errorMessage.textContent = message;
+}
```

Comment:
@JaeBeen95 

[제안]

개인 취향입니다.

```suggestion
export function setInputError(elements, message) {
  const { input, errorMessage } = elements;

  input.classList.add(SIGN_INPUT_ERROR_CLASSNAME);
  errorMessage.classList.add(ERROR_MESSAGE_CLASSNAME);
  errorMessage.textContent = message;
}
```

참고 링크
- https://hianna.tistory.com/469